legal-api/src/legal_api/models/filing.py

```diff
@@ -291,6 +294,13 @@ def get_completed_filings_for_colin():
                    Filing._status == Filing.Status.COMPLETED.value).all()
         return filings
 
+    @staticmethod
```

Comment:
consider setting some limit on this, as the next guy might just call it withA status like COMPLETED and get a few hundred thousand rows

Reply:
this is true, but it's also not necessarily incorrect/undesired behaviour...I'll chew on this